### PR TITLE
End of day 1

### PR DESCRIPTION
**Pull request by IAlibay**
I think we should make this change to the schedule.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ USA
 | 14:30-16:00   | Python Packaging                	                       | [MolSSI Python Package Set-Up Lesson](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/blob/jun24-ws/notebooks/Python_Packaging.md)  	 |
 | 16:00-16:15   | Break                            	                       |   	                                                                                                                         |
 | 16:15-18:15   | Version Control                  	                       | [MolSSI git and GitHub Lessons](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/blob/jun24-ws/notebooks/Version_Control.md)  	   |
+| 18:15         | Everyone gets a puppy | |
```
Comment (fiona-naughton) [inline README.md]: Hmm, I suggest everyone gets a kitty, instead

**Review by fiona-naughton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yuxuanzhuang) [reply README.md]: lgtm!

Comment (fiona-naughton) [reply README.md]:
```suggestion
| 18:15         | Everyone gets a kitty | |
```

File: README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ USA
 | 14:30-16:00   | Python Packaging                	                       | [MolSSI Python Package Set-Up Lesson](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/blob/jun24-ws/notebooks/Python_Packaging.md)  	 |
 | 16:00-16:15   | Break                            	                       |   	                                                                                                                         |
 | 16:15-18:15   | Version Control                  	                       | [MolSSI git and GitHub Lessons](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/blob/jun24-ws/notebooks/Version_Control.md)  	   |
+| 18:15         | Everyone gets a kitty | |
```
Comment (sjayellis) [inline README.md:43]: Where would we get the kitties?

**Review by sjayellis — COMMENTED** [review]
this is unreasonable.

**Pull request closed without merging**